test/libyul/yulOptimizerTests/rematerialiser/non_movable_function.yul

```diff
@@ -12,7 +12,7 @@
 //     let a := 1
 //     let b := f(1)
 //     let c := 1
-//     mstore(add(1, b), 1)
+//     mstore(add(1, f(1)), 1)
```

Comment:
The name of the test case suggests that the idea was for ``f`` to be non-movable and for this *not* to happen...
maybe add a ``call`` to ``f`` or something like that?